apps/modernization-ui/src/pages/patient/profile/labReport/LabReportTable.tsx

```diff
@@ -52,6 +51,22 @@ export const LabReportTable = ({ patient, pageSize, allowAdd = false }: PatientL
         }
     };
 
```

Comment:
any has to come from somewhere until transformation module is included <Maybe<AssociatedInvestigation2> does not match AssociatedInvestigation2 ?

Reply:
question:  Would the type here be `PatientLabReport | null`?

Reply:
I see the issue, the graphql schema is not strict enough and it's making the transformation difficult.

Reply:
Git hub cuts off my original comment. Maybe added on in the graphql causes the bug. 

<img width="638" alt="Screenshot 2023-12-18 at 9 08 26 AM" src="https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/3cdf3272-1ac1-4dd0-97da-127b3555386a">

Reply:
Could you add the sorter and transformer for Lab Report?

Reply:
FindLabReportsForPatient query 
does not include "content", so working on a workaround for the transformation. 

![Screenshot 2023-12-18 at 2 30 45 PM](https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/702f53e4-680c-4656-9733-5159db5fbfab)